WoWPro_Leveling/Wrath/Alliance/VAN_WOTLK_19_25_Alliance.lua

```diff
@@ -27,8 +27,8 @@ A Aquatic Form|QID|5061|C|Druid|M|56.21,30.62|Z|Moonglade|N|From Dendrite Starbl
 F Rut'theran Village|ACTIVE|5061|C|Druid|M|44.15,45.22|Z|Moonglade|N|Fly back using the Darnassus Flight Master.|
 R Darnassus|ACTIVE|5061|C|Druid|M|55.86,89.45|Z|Darnassus|N|Walk into the portal back to Darnassus.|
 T Aquatic Form|QID|5061|C|Druid|M|35.36,08.39|Z|Darnassus|N|To Mathrengyl Bearwalker.|
-H Stormwind City|AVAILABLE|244|N|Hearth to Stormwind City and rejoin the guide.|C|Druid|TZ|Stormwind City|
-F Stormwind|AVAILABLE|244|N|Fly to Stormwind.|TZ|Stormwind City|
+H Stormwind City|AVAILABLE|244|N|Hearth to Stormwind City and rejoin the guide.|C|Druid|TZ|Stormwind, Elwynn|
+F Stormwind|AVAILABLE|244|N|Fly to Stormwind.|TZ|Stormwind, Elwynn|
```

Comment:
make sure when you are chanting city names to do a /wp where. I am not entirely sure but I dont think Stormwind, Elwynn works, it should be throwing errors.

Reply:
"Stormwind City" doesn't work, that's why I changed it. The official name is "Stormwind, Elwynn"